### PR TITLE
Features

### PR DESCRIPTION
**Pull request by HammadSiddique**
## In this PR:

- [ ] Used ES6 Syntax
- [ ] Code is divided into modules which are saved in the `modules` folder
- [ ] Functionalities of the website kept without any error
- [ ] Displayed Date under navigation bar with `Luxon.js` library
- [ ] No linter Errors

### CODE REVIEW

File: modules/removebook.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+/* eslint import/prefer-default-export: "off" */
+import { Book } from './book.js';
+
+function removeBook(index) {
+  const removeBooks = new Book();
+  removeBooks.deleteBook(index);
+  window.onload = removeBooks.displayBooks();
+}
```
Comment (nicupop729) [inline modules/removebook.js]: - [x]  As we must migrate our code towards ES6 syntax, kindly refactor this function in order to use `arrow` syntax.

**Review by nicupop729 — CHANGES_REQUESTED** [review]
Hi @HammadSiddique, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
 - [X] Correct GitHub flow practice :heavy_check_mark:
 - [X] Good commit messages :heavy_check_mark:
 - [X] Professional README :heavy_check_mark:
 - [X] Linters are passing :heavy_check_mark:

## Required Changes ♻️

- [x] After adding a book, I see that is recorded in local storage, but it is not shown on the UI only after a manual refresh. Kindly, correct this for a better UI, so that after the user adds a book, immediately it will be shown on UI.
[video here](https://www.loom.com/share/a61faa060b334ce3b36628bfd8233b4d)

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by HammadSiddique** [comment]
- Modified my code according to the changes required

**Review by descholar-ceo — APPROVED** [review]
Hi @HammadSiddique , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**